packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -226,9 +232,27 @@ class _FlutterValidator extends DoctorValidator {
   }
 }
 
+class NoIdeValidator extends DoctorValidator {
+  NoIdeValidator() : super('Flutter IDE Support');
+
+  @override
+  Future<ValidationResult> validate() async {
+    // TODO(danrubel) do not show Atom once IntelliJ support is complete
```

Comment:
Atom is deprecated... only critical fixes from the Dart team; it will bit rot until it reaches some point where it is badly broken. IntelliJ is now preferred and once IntelliJ support is on par with Atom, then Atom support will be discontinued. I'm not planning to remove Atom support from `flutter doctor` any time soon, but will eventually. If the Flutter team chooses to supply resources to support Atom, then this comment does not apply and should be removed.